install/kubernetes/cilium/charts/hubble-relay/templates/deployment.yaml

```diff
@@ -27,10 +27,10 @@ spec:
             topologyKey: "kubernetes.io/hostname"
       containers:
         - name: hubble-relay
-{{- if contains "/" .Values.image }}
-          image: "{{ .Values.image }}"
+{{- if contains "/" .Values.image.repository }}
+          image: "{{ .Values.image.repository }}"
 {{- else }}
-          image: "{{ .Values.global.registry }}/{{ .Values.image }}:{{ .Values.global.tag }}"
+          image: "{{ .Values.global.registry }}/{{ .Values.image.repository }}:{{ .Values.global.tag }}"
```

Comment:
I think you meant to use `Values.image.tag` here.

Reply:
I did, thanks